core/src/controllers/rows.test.js

```diff
@@ -3,6 +3,14 @@ const supertest = require('supertest');
 const app = require('../index');
 const requestWithSupertest = supertest(app);
 
+function queryString(params) {
+  const queryString = Object.keys(params)
+    .map((key) => `${key}=${params[key]}`)
+    .join('&');
+
+  return queryString;
+}
+
```

Comment:
You can use `URLSearchParams` for this

Reply:
When I pushed the first commit, the code was using URLSearchParams, which encodes the params object and adds encoding values like %2C and others when it encounters , and space. This encoding made it difficult to check if the URL I defined in the code is equal to the response res.body.next/res.body.previous. To address this issue, I added a queryString function that does not encode the URL, allowing for easier comparison and checking.

Reply:
Aha, I see, then it's good to go man!
Thanks for another awesome PR! You're killing it!

Reply:
Thank you @thevahidal, I appreciate it